tikv-client/src/test/java/com/pingcap/tikv/predicates/ScanAnalyzerTest.java

```diff
@@ -93,7 +93,7 @@ public void buildTableScanKeyRangeTest() throws Exception {
     ScanAnalyzer scanAnalyzer = new ScanAnalyzer();
 
     List<Coprocessor.KeyRange> keyRanges =
-        scanAnalyzer.buildTableScanKeyRange(table, irs, table.getPartitionInfo());
+        scanAnalyzer.buildTableScanKeyRange(table, irs, table.getPartitionInfo().getDefs());
```

Comment:
any test for buildTableScanKeyRange when the table contains PartitionInfo?

Reply:
We do not need to. We just need to verify the correctness of pruning logic. 

There is no significant difference between the partitioned table and the normal table. If we meet a partitioned table, we only need to replace the table id with partition id.

Reply:
Then you may comment this :)